shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/statement/dml/SelectStatementContext.java

```diff
@@ -128,15 +128,17 @@ private Map<Integer, SelectStatementContext> createSubqueryContexts(final Map<St
     }
     
     private Map<String, ShardingSphereSchema> getSchemas(final Map<String, ShardingSphereDatabase> databases, final String databaseName) {
-        ShardingSphereDatabase database = databases.get(databaseName);
-        if (null == database) {
+        if (null == databaseName) {
             if (tablesContext.getTables().isEmpty()) {
                 return Collections.emptyMap();
             } else {
-                throw new DatabaseNotExistedException(databaseName);
+                throw new DatabaseNotExistedException(null);
```

Comment:
null parameter maybe not good, how about create a new exception?
maybe `NoDatabaseException` is a good name.

Reply:
Yeah, i agree. I will adjustment it